app/repositories/enrolment_repo.py

```diff
@@ -0,0 +1,22 @@
+import abc
+from typing import List
+
+from app.domain.entities.enrolment_request import EnrolmentFilters, EnrolmentRequest
+
+
+class EnrolmentRepo(abc.ABC):
+    @abc.abstractmethod
+    def get_enrolments(
+        self, enrolment_filters: EnrolmentFilters
+    ) -> List[EnrolmentRequest]:
+        """"""
+
+    @abc.abstractmethod
+    def get_enrolment_by_id(self, enrolment_request_id: str) -> EnrolmentRequest:
```

Comment:
better if get_enrl_request? because enrl is different.

Reply:
Changed to: get_enrolment_authorisation_by_id